finish/Dockerfile

```diff
@@ -3,21 +3,34 @@
 FROM openliberty/open-liberty:kernel-java8-openj9-ubi
 # end::from[]
 
+ARG VERSION=1.0
+ARG REVISION=SNAPSHOT
+# tag::label[]
+
+LABEL \
+  org.opencontainers.image.authors="Your Name" \
+  org.opencontainers.image.vendor="IBM" \
+  org.opencontainers.image.url="local" \
+  org.opencontainers.image.source="https://github.com/OpenLiberty/guide-docker" \
+  org.opencontainers.image.version="$VERSION" \
+  org.opencontainers.image.revision="$REVISION" \
+  vendor="Open Liberty" \
+  name="system" \
+  version="$VERSION-$REVISION" \
+  summary="The system microservice from the Docker Guide" \
+  # tag::description[]
+  description="This image contains the system microservice running with the Open Liberty runtime."
+  # end::description[]
+# end::label[]
+
 # tag::user-root[]
 USER root
 # end::user-root[]
-# Symlink servers directory for easier mounts.
-# tag::link[]
-RUN ln -s /opt/ol/wlp/usr/servers /servers
-# end::link[]
+
+# tag::copy[]
+COPY --chown=1001:0 src/main/liberty/config/ /config/
```

Comment:
For devc, we recommend using COPY with specific files in the config directory since this allows dev mode to track its changes for hot deployment without needing to rebuild the image.  See [point 1 in the devc file tracking section](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#file-tracking).  
e.g. change this line to:
```
COPY --chown=1001:0 src/main/liberty/config/server.xml /config/
```
And update the relevant section of the README.adoc to correspond to this.

Reply:
Thanks for the feedback.  Your suggested changes have been made.